README.md

```diff
@@ -39,7 +39,7 @@ https://developer.android.com/studio/build/maven-publish-plugin
 
 The plugin applies & configures the `maven-publish` plugin by adding the S3 repository as well as a `MavenPublication` that uses the calculated version name. It provides 3 Gradle tasks:
 
-* `publishToS3` takes `--branch-name`, `--sha1`, `--tag-name` & `--pull-request-url` command line arguments and uses the information to calculate a version name, verify that this version is not already published to S3 and finally calls `publishS3PublicationToS3Repository` gradle task which will publish the artifacts to S3.
+* `publishLibraryToS3` takes `--branch-name`, `--sha1`, `--tag-name` & `--pull-request-url` command line arguments and uses the information to calculate a version name, verify that this version is not already published to S3 and finally calls `publishS3PublicationToS3Repository` gradle task which will publish the artifacts to S3.
```

Comment:
~Unsure why we need all of `--branch-name`, `--sha1` AND `--tag-name`, can't we infer one from the other?
Or if those arguments are supposed to be exclusive (i.e. you're supposed to use only one of them depending on what you want to achieve), might be worth making that clearer?~

nvm, I see when I read beyond just the diff and also check what's below in the file, that this is explained further down. Still maybe adapt the wording?

```suggestion
* `publishLibraryToS3` can take a mix of the following command line arguments: `--branch-name`, `--sha1`, `--tag-name`, `--pull-request-url`. Depending on the arguments passed, it uses the information to calculate a version name (see below), verify that this version is not already published to S3 and finally calls `publishS3PublicationToS3Repository` gradle task which will publish the artifacts to S3.
```

Reply:
I like the wording you've suggested and applied it directly in dab5425836e808702af8e9a38c81fc990d91432e.